### DISCUSSION TITLE
How to selectively answer incoming calls

### DISCUSSION THREAD

**Discussion by abj3ct10n** [baresip/baresip#2530] [Q&A]
Hello, I checked the relevant documents and code. I thought I would use the `/acceptdir audio=sendrecv video=inactive 2222@192.168.1.111` approach to select 2222 incoming calls.

My current SIP Online is as follows:
```
2222@192.168.1.111
3333@192.168.1.111
```

They will be online at the same time and will support simultaneous phone calls.
How to selectively answer a call when they both have incoming calls. For example, if I only answer calls from 2222.192.168.1.111, I will reject calls from 3333@192.168.1.111.

Unfortunately, I checked the documentation and failed to receive calls from 2222.192.168.111 using
`/acceptdir audio=sendrecv video=inactive 2222@192.168.111`

Thank you very much for your help. Thank you.

**Comment by abj3ct10n** [comment]
Ok, so I'm going to look up the call status by `l` , use `/accept [ID]` to specify the answer, and use `/hangup [ID]` to specify the hang-up, which I know so far.

I think it may need to be mentioned in the document. I'm sorry to disturb you. The problem has been solved.

Thank you very much

**Comment by juha-h** [comment]
@abj3ct10n If the problem is solved, please close the issue.
